packages/bits/demo/src/components/demo/wizard-v2/wizard-custom-layout/wizard-custom.component.less

```diff
@@ -23,7 +28,11 @@
         color: @nui-color-text-light;
         border-color: transparent;
 
-        svg path {
+        .nui-wizard-step-label{
+            color: @nui-color-text-inverse;
+        }
+
+        .nui-step-icon-selected svg circle, .nui-step-icon-selected svg path {
             fill: @nui-color-text-inverse;
```

Comment:
These color's aren't dark mode friendly. To make them dark mode friendly, we should use the setCssVariable mixin.